From aaa281dc330b5bbeb0878f67cd0e7f29d7bdf937 Mon Sep 17 00:00:00 2001
From: Demon Singur <demonsingur@gmail.com>
Date: Sat, 19 May 2018 13:16:04 +0000
Subject: [PATCH 359/420] {chiron,sagit}_defconfig: enable qcacld-3.0

All the configs from stock, except
CONFIG_WLAN_OFFLOAD_PACKETS=y
which didn't seem to be enabled in 8.4.19's qcacld.ko.

Change-Id: I1da7e60dd4a14c3abdd0c32b2a2bb8505b17b810
---
 arch/arm64/configs/chiron_defconfig | 24 ++++++++++++++++++++++++
 arch/arm64/configs/sagit_defconfig  | 24 ++++++++++++++++++++++++
 2 files changed, 48 insertions(+)

diff --git a/arch/arm64/configs/chiron_defconfig b/arch/arm64/configs/chiron_defconfig
index 2d26e1c11d2f..26799f738c91 100644
--- a/arch/arm64/configs/chiron_defconfig
+++ b/arch/arm64/configs/chiron_defconfig
@@ -519,6 +519,30 @@ CONFIG_ANDROID_TIMED_GPIO=y
 CONFIG_ANDROID_LOW_MEMORY_KILLER=y
 CONFIG_ION=y
 CONFIG_ION_MSM=y
+CONFIG_QCA_CLD_WLAN=y
+CONFIG_QCACLD_WLAN_LFR3=y
+CONFIG_WLAN_FEATURE_11W=y
+CONFIG_WLAN_FEATURE_LPSS=y
+CONFIG_QCOM_VOWIFI_11R=y
+CONFIG_QCACLD_FEATURE_NAN=y
+CONFIG_WLAN_FEATURE_NAN_DATAPATH=y
+CONFIG_QCACLD_FEATURE_GREEN_AP=y
+CONFIG_HELIUMPLUS=y
+CONFIG_64BIT_PADDR=y
+CONFIG_QCOM_TDLS=y
+CONFIG_QCOM_LTE_COEX=y
+CONFIG_MPC_UT_FRAMEWORK=y
+CONFIG_WLAN_OFFLOAD_PACKETS=y
+CONFIG_FEATURE_TSO=y
+CONFIG_WLAN_FASTPATH=y
+CONFIG_WLAN_NAPI=y
+CONFIG_WLAN_TX_FLOW_CONTROL_V2=y
+CONFIG_WLAN_SYNC_TSF=y
+CONFIG_LFR_SUBNET_DETECTION=y
+CONFIG_MCC_TO_SCC_SWITCH=y
+CONFIG_QCACLD_WLAN_LFR2=y
+CONFIG_WLAN_FEATURE_DISA=y
+CONFIG_WLAN_FEATURE_FILS=y
 CONFIG_QPNP_REVID=y
 CONFIG_QPNP_COINCELL=y
 CONFIG_SPS=y
diff --git a/arch/arm64/configs/sagit_defconfig b/arch/arm64/configs/sagit_defconfig
index 30a10d6f863b..5f1f5e6c13dc 100644
--- a/arch/arm64/configs/sagit_defconfig
+++ b/arch/arm64/configs/sagit_defconfig
@@ -519,6 +519,30 @@ CONFIG_ANDROID_TIMED_GPIO=y
 CONFIG_ANDROID_LOW_MEMORY_KILLER=y
 CONFIG_ION=y
 CONFIG_ION_MSM=y
+CONFIG_QCA_CLD_WLAN=y
+CONFIG_QCACLD_WLAN_LFR3=y
+CONFIG_WLAN_FEATURE_11W=y
+CONFIG_WLAN_FEATURE_LPSS=y
+CONFIG_QCOM_VOWIFI_11R=y
+CONFIG_QCACLD_FEATURE_NAN=y
+CONFIG_WLAN_FEATURE_NAN_DATAPATH=y
+CONFIG_QCACLD_FEATURE_GREEN_AP=y
+CONFIG_HELIUMPLUS=y
+CONFIG_64BIT_PADDR=y
+CONFIG_QCOM_TDLS=y
+CONFIG_QCOM_LTE_COEX=y
+CONFIG_MPC_UT_FRAMEWORK=y
+CONFIG_WLAN_OFFLOAD_PACKETS=y
+CONFIG_FEATURE_TSO=y
+CONFIG_WLAN_FASTPATH=y
+CONFIG_WLAN_NAPI=y
+CONFIG_WLAN_TX_FLOW_CONTROL_V2=y
+CONFIG_WLAN_SYNC_TSF=y
+CONFIG_LFR_SUBNET_DETECTION=y
+CONFIG_MCC_TO_SCC_SWITCH=y
+CONFIG_QCACLD_WLAN_LFR2=y
+CONFIG_WLAN_FEATURE_DISA=y
+CONFIG_WLAN_FEATURE_FILS=y
 CONFIG_QPNP_REVID=y
 CONFIG_QPNP_COINCELL=y
 CONFIG_SPS=y
-- 
2.15.0

